edkrepo_manifest_parser/edk_manifest.py

```diff
@@ -140,6 +144,8 @@ def __init__(self, fileref):
         self._folder_to_folder_mappings = []  # List of FolderToFolderMapping objects
         self._submodule_alternate_remotes = []
         self._submodule_init_list = []
+        self._patch_sets = {} # A dict of patchsets with their name as key
+        self._patch_set_operations = defaultdict(list) # A list of patchSet operations with name as key

```

Comment:
The comment here seems refers to the list value being passed into the default dict constructor not the contents of self.__patch_set_operations which is actually a dict with the patchset name as the key and the data being a list of patch set operations. 

Can you provide some context behind your decision to use a default dict instead of a standard dict?